Header header
uint8 RUN_MISSION = 0
uint8 ABORT = 1
uint8 ABORT_TO_HOME = 2
uint8 ABORT_TO_START_POS = 3